static_report/src/components/ComparisonReport/CRModal/CRModalPiperiderTable.tsx

```diff
@@ -11,7 +11,7 @@ import { formatTestExpectedOrActual } from '../../../utils/formatters';
 import { TestStatus } from '../../shared/TestStatus';
 import { CRModalData } from './CRModal';
 
-type Props = { data: CRModalData };
+type Props = { data?: CRModalData };
```

Comment:
Do you want to change the name like below?

Reply:
Do you mean `<name>Props`? 
if so, i think rule of thumb is to use generic `Props` unless you have multiple components in a file that have different `Props` namespaces. 

Also, since we aren't `export`ing the props don't need worry about naming collision with other exported members

Reply:
That makes sense, I agreed with you.